### PR TITLE
RFC 141: Change testharness to include late tests

### PR DESCRIPTION
**Pull request by jcscottiii**
[Rendered](https://github.com/web-platform-tests/rfcs/blob/include_late_tests/rfcs/testharness_include_late_tests.md)

### CODE REVIEW

**Comment by jcscottiii** [comment]
Linked implementation: https://github.com/web-platform-tests/wpt/pull/38806

**Review by Ms2ger — APPROVED** [review]
This seems reasonable.

**Review by gsnedders — APPROVED** [review]

File: rfcs/testharness_include_late_tests.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+Currently, "late tests" are
+[ignored](https://github.com/web-platform-tests/wpt/blob/c751476c04678faa79e598683f2a6e94d17cf9e0/resources/test/tests/unit/late-test.html#L10-L17).
+This RFC proposes that testharness should now include them.
+
+## Details
+
+Conside the following test:
+
+```
+\<script>
```
Comment (foolip) [inline rfcs/testharness_include_late_tests.md:14]: The `\` ends up in the rendered input. But you can simply drop the HTML and make this a pure JS example:

```js
window.onload = () => {
  test((t) => { ... }, 'test 1');
  test((t) => { ... }, 'test 2');
  test((t) => { ... }, 'test 3');
};
```

File: rfcs/testharness_include_late_tests.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+window.onload = () => {
+  test((t) => { ... }, 'test 1');
+  test((t) => { ... }, 'test 2');
+  test((t) => { ... }, 'test 3');
+};
+\</script>
+```
+
+Currently, only test 1 will be run. The issue is that the testharness
+immediately adds a window load handler that marks all_loaded = true,
```
Comment (foolip) [inline rfcs/testharness_include_late_tests.md:24]: Can you sprinkle some backticks around here to make the code formatted `like this`?

**Review by foolip — APPROVED** [review]

**Pull request merged**